vllm/model_executor/layers/quantization/utils/fp8_utils.py

```diff
@@ -81,6 +82,25 @@ def apply_w8a8_block_fp8_linear(
     return output.to(dtype=input.dtype).view(*output_shape)
 
 
+def apply_w8a8_block_fp8_linear_fake(
+    input: torch.Tensor,
+    weight: torch.Tensor,
+    block_size: List[int],
+    weight_scale: torch.Tensor,
+    input_scale: Optional[torch.Tensor] = None,
+) -> torch.Tensor:
+    output_shape = [*input.shape[:-1], weight.shape[0]]
+    return torch.empty_like(input, shape=output_shape)
```

Comment:
```torch.empty_like``` doesn't take a ```shape``` argument. We may want to use 

```
return torch.empty(output_shape, dtype=input.dtype, device=input.device)
```

Reply:
Oops. Fixed. Thanks!